src/components/ProfilePage.jsx

```diff
@@ -1,9 +1,20 @@
 import NavBar from './NavBar'
 import { Card, ListGroup, ListGroupItem, Row } from 'react-bootstrap'
-import profile from '../images/profile.jpg'
-import ItemContainer from './ItemContainer'
+import profile from '../images/profile.png'
+import { useDispatch, useSelector } from 'react-redux'
+import { React, useEffect } from 'react'
+import { getCurrentUserAsync } from '../redux/users/thunks'
 
 function ProfilePage () {
+  const userId = useSelector(state => state.user.currentUserId)
+  const dispatch = useDispatch()
+  useEffect(() => {
+    dispatch(getCurrentUserAsync(userId))
+  }, [dispatch])
+
+  const currentUser = useSelector(state => state.user.currentUser)
+  console.log('=>', currentUser)
```

Comment:
did you mean to leave this `console.log` in?

Reply:
ah for now yes, I'll remove it when the PR is completed (I've updated my PR heading with the WIP tag)